dpcpp/test/base/executor.dp.cpp

```diff
@@ -208,30 +200,6 @@ TEST_F(DpcppExecutor, CopiesDataToCPU)
     dpcpp->free(copy);
 }
 
-
-TEST_F(DpcppExecutor, CopiesDataToGPU)
```

Comment:
Should we keep this?
creating executor with arg "cpu" or "gpu" ignore the SYCCL_DEVICE_TYPE